README.md

```diff
@@ -0,0 +1,218 @@
+<a name="readme-top"></a>
+<div align="center">
+    <img src="./logo.png">
+</div>
+
+<div align="center">
+
+  <h2><b>World Vision Clinic</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 World Vision Clinic ](#-world-vision-clinic-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [Front-end repository link](#front-end-repository-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Check Linter](#check-linter)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 World Vision Clinic <a name="about-project"></a>
+
+This is a full-stack project. In which you can register yourself create a doctor, book an appointment
+with the docotor, see all appiontmnets, delete a doctor(s).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User can register himself.
+- Add a doctor.
+- Book an appointment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+### 🚀 Live Demo <a name="live-demo"></a>
+
+- [API on Render](https://appoint-doctor.onrender.com/)
+
+## Front-end repository link
+
+- <a href="https://github.com/MudasirSherwani/Book-Appointment-Front-End" target="_blank">React Front-end</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - PostgreSQL.
+ - Vs Code.
+
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+ - mkdir my-folder
+ - cd my-folder
+ - git clone git@github.com:MudasirSherwani/Book-an-Appointment.git
+ - cd Book-an-Appointment
+ - bundle install
+ - db:create
+ - db:migrate
+
+```
+### Usage
+
+To run the project, execute the following command:
+
+- rails server
+
+### Run tests
+
+- rspec
```

Comment:
- [ ] This `rspec` command alone will deal a lot of failed tests:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/9a77da34-23b0-4c86-afc7-dde4f6438289)

## Tips and extra help

This is because it is including the swagger integration tests.  Those will fail without the proper command.

You'll need to be more specific in your instructions:

```bash
rspec  ./spec/models/
```

Result:

![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/57421823/a820fac5-b83e-4204-8c2e-95cbd92f83af)

That way, only the unit tests will be run, and all of them will pass. 💪 

Kindly update your readme instructions using the example above.